docs/user-guide/cloud-with-mirror.md

```diff
@@ -0,0 +1,241 @@
+
+* quay.io/edge-infrastructure/assisted-installer-agent:latest
+* quay.io/edge-infrastructure/assisted-installer:latest
+* quay.io/edge-infrastructure/assisted-installer-controller:latest
+
+The Assisted Installer service hosted at [console.redhat.com](https://console.redhat.com) does not support using mirrored instances of the above container images at this time.
+
+If you use the "minimal" ISO image to boot your machines, each host will pull an ISO image from [mirror.openshift.com](https://mirror.openshift.com) in order to complete the inital boot process. It is suggested to use the full image to keep this from occuring.
+
+## Requirements
+
+* A container registry to mirror an OpenShift release
+* curl
+* jq
+* sed
+* base64
+
+## Identify a Container Registry and Mirror Contents
+
+You will need to have a container registry available in the disconnected environment to house all the container images that are required to complete an OpenShift install. See [Mirroring image for a disconnected installation](https://docs.okd.io/latest/installing/disconnected_install/installing-mirroring-installation-images.html) or [Mirroring image for a disconnected installation using the oc-mirror plug-in](https://docs.okd.io/latest/installing/disconnected_install/installing-mirroring-disconnected.html) for how to mirror the required images. 
+
+If you do not have a container registry available you, instructions for installing a container registry and mirroring the contents into that registry can be found in [Creating a mirror registry with mirror registry for Red Hat OpenShift](https://docs.okd.io/latest/installing/disconnected_install/installing-mirroring-creating-registry.html).
+
+Once you have mirrored the OpenShift container images, and the [Operator Hub catalog](https://docs.okd.io/latest/installing/disconnected_install/installing-mirroring-installation-images.html#olm-mirror-catalog_installing-mirroring-installation-images) you can move onto the next steps.
+
+> **NOTE**: You must ensure that you have mirrored a copy of OCP that _EXACTLY_ matches the version that you are installing from the Assisted Installer service hosted at [console.redhat.com](https://console.redhat.com). eg. If you are installing OpenShift 4.11.5 from the Assisted Installer, you must have previously mirrored verion 4.11.5 to your local registry.
+
+## Environment Setup
+
+### User Authentication
+
+User Authentication is using JWT tokens. The JWT token is **valid for 15 minutes** and needs to be provided in the header of the HTTP request.
+
+First you must obtain your offline token from https://console.redhat.com/openshift/token (This token does not expire, so do not share it with others) and set it as OFFLINE_TOKEN, then generate your JWT_TOKEN:
```

Comment:
This is repeating documentation we already have in cloud.md

Reply:
@omertuc thanks for the feedback, I have updated this section and removed the duplicate documentation. Please let me know if you have any additional questions about this.